geps/gep-1897/index.md

```diff
@@ -396,35 +721,44 @@ developer to provide the appropriate configuration for TLS.  The implementation
-adding hostname for SNI and removing allowed SANs.
+A. This was most recently addressed by adding hostname for SNI and removing allowed SANs.
+
+Q. Rob Scott is interested in extending the TargetRef to optionally include port, since we are targeting the entirety
+of a Service. See the discussion in https://github.com/kubernetes-sigs/gateway-api/pull/2113/files#r1231594914,
+and follow-up issue in https://github.com/kubernetes-sigs/gateway-api/issues/2147.
+
+A. TargetRef has been changed to `LocalPolicyTargetReferenceWithSectionName`, wherein the `SectionName` field is
+interpreted as a port name for a Service.
 
 ## Graduation Criteria
 
 This section is to record issues that were requested for discussion in the API section before this GEP graduates
 out of `Provisional` status.
 
-1. Rob Scott is interested in extending the TargetRef to optionally include port, since we are targeting the entirety
-of a Service. See the discussion in https://github.com/kubernetes-sigs/gateway-api/pull/2113/files#r1231594914,
-and follow up issue in https://github.com/kubernetes-sigs/gateway-api/issues/2147
-2. Michael Pleshakov asked about conflicts that could arise when multiple implementations are running in a cluster.
+1. Michael Pleshakov asked about conflicts that could arise when multiple implementations are running in a cluster.
 This is a gap in our policy attachment model that needs to be addressed.  See the discussion in
 https://github.com/kubernetes-sigs/gateway-api/pull/2113/files#r1235750540. Graduating this GEP to implementable
 requires an update to the Policy GEP to define how status can be nested to support multiple implementations. This will
 likely look very similar to Route status.
-See [comment](https://github.com/kubernetes-sigs/gateway-api/pull/2113#issuecomment-1696127092).
-3. Rob Scott [wanted to note](https://github.com/kubernetes-sigs/gateway-api/pull/2113#issuecomment-1696127092) that
+
+>This question has been converted to a Gateway API enhancement request for Policy:
+https://github.com/kubernetes-sigs/gateway-api/issues/4098.
+
+2. Rob Scott [wanted to note](https://github.com/kubernetes-sigs/gateway-api/pull/2113#issuecomment-1696127092) that
 when this graduates to the standard channel, implementations of HTTPRoute may also be
 required to watch the BackendTLSPolicy. If one of these policies is attached to a Service targeted by an HTTPRoute,
 the implementation would be required to fully implement the policy or mark the backend invalid.
 
+>This comment may be added to the release notes for v1.4.0 of Gateway API, along with other special notes for the introduction
+of the first standard Policy type.
+
 ## References
 
 [Gateway API TLS Use Cases](https://docs.google.com/document/d/17sctu2uMJtHmJTGtBi_awGB0YzoCLodtR6rUNmKMCs8/edit#heading=h.cxuq8vo8pcxm)
 
 [GEP-713: Metaresources and PolicyAttachment](../gep-713/index.md)
 
-[Gateway API TLS](../../guides/tls.md)
+[Gateway API TLS](../../site-src/guides/tls.md)
```

Comment:
this is broken on the preview:

https://deploy-preview-4099--kubernetes-sigs-gateway-api.netlify.app/geps/gep-1897/